react/css/src/digitv2/pages/employee/boundaryFilter.scss

```diff
@@ -0,0 +1,58 @@
+.selecting-boundary-div {
```

Comment:
@abishekTa-egov Make sure you don't update any of the existing classes, you have to add new classes for your use cases so it doesn't cause any issue for others